### DISCUSSION TITLE
Codepen console can be misleading

### DISCUSSION THREAD

**Discussion by markfirmware** [alpinejs/alpine#982] [4. General]
https://codepen.io/markfirmware/pen/gOwzreB

The x-if probably produces an error message due to defective alpine error processing code itself. The message makes no sense.

**Comment by ryangjchandler** [comment]
@markfirmware Please be more helpful and provide details about the problems you're experiencing.

**Reply by markfirmware** [reply]
I mistook the codepen console claiming an alpine error for a badly constructed error from alpine itself.

**Comment by KevinBatdorf** [comment]
Keep in mind when you post a discussion here that you're sending a notification to 200 people that monitor these discussions to provide help and also keep up to date with genuine issues with the framework. 

Put some thought into how you title your discussions (this discussion title was perhaps not very good?) and put some effort into the descriptions, the issues, your suggestion solution, what you've tried, etc.

Otherwise there's a Discord that's more suited for quick chat and some flexibility for users to have some friendly banter and go back and forth a bit on issues, etc. Plenty of people chatting there.

**Reply by markfirmware** [reply]
Thanks for letting me fix the title myself - you have responded to discourtesy with courtesy.
